extensions/amp-date-countdown/1.0/web-component.js

```diff
@@ -0,0 +1,32 @@
+import {defineBentoElement} from '#preact/bento-ce';
+
+import {getTemplate} from '#utils/template-utils';
+
+import {BaseElement, TAG} from './base-element';
+
+import {mustache} from '../../amp-mustache/1.0/bento-mustache';
```

Comment:
Let's place this in `src/bento/components`. 

Let's also make it a default export so that we can directly remap to `mustache` on the NPM package.

```suggestion
import mustache from '#bento/components/bento-mustache/1.0/bento-mustache';
```